nullaway/src/main/java/com/uber/nullaway/CodeAnnotationInfo.java

```diff
@@ -123,6 +123,16 @@ public boolean isSymbolUnannotated(Symbol symbol, Config config) {
     Symbol.ClassSymbol classSymbol;
     if (symbol instanceof Symbol.ClassSymbol) {
       classSymbol = (Symbol.ClassSymbol) symbol;
+    } else if (symbol.name.contentEquals("class")
+        && symbol.owner.getKind().equals(ElementKind.CLASS)
+        && symbol.owner.getQualifiedName().equals(symbol.owner.getSimpleName())
+        && symbol.owner.enclClass() == null) {
```

Comment:
Extract condition to a method, like `isClassFieldOfPrimitiveType(symbol)`?  This is optional